src/test/java/com/hotels/mutantswarm/MutantSwarmTest.java

```diff
@@ -22,13 +22,18 @@
 import java.util.List;
 
 import org.junit.Test;
+import org.junit.jupiter.api.TestTemplate;
+import org.junit.jupiter.api.extension.ExtendWith;
 import org.junit.runner.RunWith;
 
 import com.klarna.hiverunner.HiveShell;
 import com.klarna.hiverunner.annotations.HiveSQL;
 import com.klarna.hiverunner.annotations.HiveSetupScript;
 
-@RunWith(MutantSwarmRunner.class)
+//To run with Junit4, replace '@ExtendWith(MutantSwarmTestExtension.class)' with '@RunWith(MutantSwarmRunner.class)' and '@TestTemplate' with '@Test'
```

Comment:
We should have two test classes - one using the runner and another using the extension, to prove that both work.